manual.asc

```diff
@@ -937,18 +937,24 @@ To stop receiving notifications go to Deck options > Reminders and unmark the ch
 
 [[betaTesting]]
 === Beta testing
-If you want to try out the latest features in AnkiDroid, you can sign up for the beta testing program as follows:
+
+Beta releases :: If you want to try out the latest features in AnkiDroid, you can sign up for the beta testing program as follows:
 
  . Visit the https://play.google.com/apps/testing/com.ichi2.anki[Google Play Beta page]
  . Click *Become a beta tester*
  
-After following these steps, the latest beta version will automatically be installed by Google Play in the same way as ordinary updates.
-If you are more adventurous, you can also become an alpha tester, by joining the https://groups.google.com/forum/#!forum/ankidroidalphatesters[alpha testers group]
-in addition to performing the above steps for beta testing.
+Alpha releases :: If you are more adventurous, you can also become an alpha tester, by joining the https://groups.google.com/forum/#!forum/ankidroidalphatesters[alpha testers group]
+in addition to performing the above steps for beta testing. The group itself receives no messages, it is used exclusively to determine which Google accounts Google Play will deliver alpha releases to.
+
+Beta / alpha / stable release compatibility :: The 2.8.x releases and the 2.9.x releases are forwards and backwards compatible, with no known risk of data loss. It should be safe to try a 2.9.x alpha or beta while preserving the ability to leave the program and use 2.8.x on the same collection.
```

Comment:
This information about 2.8.x / 2.9.x is too specific for the manual, I prefer to omit it.

Reply:
Dang - this exact info (the compatibility, and the following about how to get it) is a thing I either want to explain to people, or clarifies something vague. Is there any wording you would be willing to accept that tells people the current alphas are backwards-compatible, and how to get it immediately?